query_tests/generate_rnd_queries.py

```diff
@@ -442,7 +469,26 @@ def generate_query(data_faker, columns, schema, table):
             expressions.append(random.choice(CONJUNCTIONS))
         expressions.append(expr)
     filter_ = ' '.join(expressions)
-    return f'SELECT count(*) FROM "{schema}"."{table}" WHERE {filter_};'
+    use_window_functions = every(20)
+    wfunction_expr = None
+    if use_window_functions:
+        wfunction = random.choice(WINDOW_FUNCTIONS)
+        if wfunction:
+            column = get_number_or_text_column(columns)
+            wfunction_expr = wfunction(data_faker, column)
+    if wfunction_expr is not None:
+        return f'SELECT {wfunction_expr} FROM "{schema}"."{table}" WHERE {filter_};'
```

Comment:
This could probably be moved into the previous if block to remove one more `if` branch. (Unless `wfFunction` can return None)